### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by dennisklein**
Bunch of small commits cluttering up my private dev branch that are ready to be upstreamed.

### CODE REVIEW

**Comment by dennisklein** [comment]
1. `build/fedora-32-x86_64-gcc-10` and `build/ubuntu-20.04-x86_64-gcc-9` are understood, will fix them tomorrow
2. `build/fedora-34-x86_64-gcc-11` is not yet understood (https://cdash.gsi.de/testDetails.php?test=8539313&build=278626, [dds_2021-07-02.0.log](https://github.com/FairRootGroup/FairMQ/files/6751374/dds_2021-07-02.0.log)) - need to catch it with higher log level
3. `check/static-analyzers` looks like the same as (2)
4. sanitizers show multiple different issues, need to look in more details tomorrow.

**Comment by dennisklein** [comment]
Removed thread and memory sanitizer checks again. In order to get rid of the false positives we have to compile the whole stack with `-fsanitize` including `libstdc++`. This is a task for another day.

File: examples/dds/fairmq-start-ex-dds.sh.in [hunk 1]
```diff
@@ -79,6 +79,6 @@ if dds-agent-cmd getlog -a; then
 fi
 
 # This string is used by ctest to detect success
-echo "Example successful :)"
```
Comment (rbx) [inline examples/dds/fairmq-start-ex-dds.sh.in:88]: :(

File: examples/dds/fairmq-start-ex-dds.sh.in [hunk 2]
```diff
@@ -79,6 +79,6 @@ if dds-agent-cmd getlog -a; then
...
+echo "Example successful"
```
Comment (rbx) [inline examples/dds/fairmq-start-ex-dds.sh.in:82]: :(

**Review by rbx — APPROVED** [review]

**Pull request merged**